tests/ftests/051-sudo-cgroup_get_cgroup.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python3
+# SPDX-License-Identifier: LGPL-2.1-only
+#
+# Exercise cgroup_create_cgroup() and cgroup_get_cgroup()
+#
+# Copyright (c) 2022 Oracle and/or its affiliates.
```

Comment:
I  understand some of the test cases were written in  2022, should the year get updated to 2023?

Reply:
Yep